test/test_tensor.py

```diff
@@ -109,6 +109,19 @@ def test_dropout(self):
     non_zeros = np.count_nonzero(w.cpu().numpy())
     expected = n * (1 - rate)
     np.testing.assert_allclose(non_zeros, expected, rtol=2e-3)
+    
+  def test_arange(self):
+    arange = Tensor.arange(stop=1337)
+    expected = Tensor(np.arange(stop=1337))
+    assert type(arange) is Tensor
+    np.testing.assert_allclose(arange.numpy(), expected.numpy())
```

Comment:
I think we already have an arange test in `test/test_ops.py`

Maybe move the `cumsum` test there as well?

Reply:
Sure, moved the cumsum test to test_ops.
All tests passed.